### ISSUE TITLE
Real data in Ecommerce site

### ISSUE THREAD

**Issue by SwatiMe** [swarnavopramanik/Admin-Panel#5]
Hii, I have observed that u have used dummy data in your Ecommerce website. If you allow me I can connect it to real data through mongo db under Gssoc2023. Please grant me the access of code.

**Comment by swarnavopramanik** [comment]
Yes sure you work but make sure if it will run your local machine then you pull request send otherwise please don't

**Comment by AKsHaT123456A** [comment]
Hi @swarnavopramanik should I too work on it?

**Comment by swarnavopramanik** [comment]
Yeahh sure 😀

**Comment by AKsHaT123456A** [comment]
Hi @swarnavopramanik should i use my own database key?

**Comment by swarnavopramanik** [comment]
Yeahh sure but share it public because it is  opesource project not include .env file

**Comment by AKsHaT123456A** [comment]
Fine will give you updates as soon as possible

**Comment by swarnavopramanik** [comment]
Yeah sure

**Comment by AKsHaT123456A** [comment]
Hi @swarnavopramanik i have added the cartData in the backend working on others too,but i had a question , i don't have to store the functions using props...right? As they are frontend rendering functions.And also if possible kindly assign me this issue..

**Comment by swarnavopramanik** [comment]
> Hi @swarnavopramanik i have added the cartData in the backend working on others too,but i had a question , i don't have to store the functions using props...right? As they are frontend rendering functions.And also if possible kindly assign me this issue..
>  

Yeahh you try this and if work then take ss then send me then you merge please

**Comment by AKsHaT123456A** [comment]
> > Hi @swarnavopramanik i have added the cartData in the backend working on others too,but i had a question , i don't have to store the functions using props...right? As they are frontend rendering functions.And also if possible kindly assign me this issue..
> 
> Yeahh you try this and if work then take ss then send me then you merge please

Also can you assign me this issue too?

**Comment by swarnavopramanik** [comment]
> > > Hi @swarnavopramanik i have added the cartData in the backend working on others too,but i had a question , i don't have to store the functions using props...right? As they are frontend rendering functions.And also if possible kindly assign me this issue..
> > 
> > Yeahh you try this and if work then take ss then send me then you merge please
> 
> Also can you please assign me this issue too?
> 

Yeahh sure please start work 
I will add GSSoC level tommorow don't worry

**Comment by AKsHaT123456A** [comment]
ok

> > > > Hi @swarnavopramanik i have added the cartData in the backend working on others too,but i had a question , i don't have to store the functions using props...right? As they are frontend rendering functions.And also if possible kindly assign me this issue..
> > > 
> > > 
> > > Yeahh you try this and if work then take ss then send me then you merge please
> > 
> > 
> > Also can you please assign me this issue too?
> 
> Yeahh sure please start work I will add GSSoC level tommorow don't worry

Ok already started working on it

**Comment by AKsHaT123456A** [comment]
Hi @swarnavopramanik about the photos , shall I create a drive link and store them or i should store them directly as buffer data?

**Comment by swarnavopramanik** [comment]
> Hi @swarnavopramanik about the photos , shall I create a drive link and store them or i should store them directly as buffer data?
> Sorry for disturbing you this much

Store as a buffer data in mongo db with the help of mongo campus

**Comment by mohd-raza** [comment]
@SwatiMe @AKsHaT123456A Any updates guys??? Any way to connect through database?

**Comment by AKsHaT123456A** [comment]
@mohd-raza trying to work on it the buffer format is troubling so trying to sort that out
